Runar/Runar/Controllers/Generator flow/GenerationPopUpViewController.swift

```diff
@@ -102,13 +102,17 @@ public class GenerationPopUpViewController: UIViewController {
     public func setupView(view: UIView) {
         self.runeView = view
     }
-    
-    public func setupModel(_ model: GenerationRuneModel?) {
+
+    public func setupModel(_ model: GeneratorCoreData?) {
         self.runeModel = model
-        
-        self.imageView.image = model?.image.image
-        self.header.attributedText = UILabel.getAttributedText(text: model!.title, lineHeight: 0.7)
-        self.desc.attributedText = UILabel.getAttributedText(text: model!.description, lineHeight: 1.12)
+
+        guard let imageData = model?.runeImage?.image else { return }
```

Comment:
imageData or image?

Reply:
First get imageData with Data type, then use it to get image